contrib/extprotocol/gpextprotocol.c

```diff
@@ -104,8 +104,10 @@ demoprot_import(PG_FUNCTION_ARGS)
 		myData->filename = pstrdup(parsed_url->path);
 	
 		if(strcasecmp(parsed_url->protocol, p_name) != 0)
-			elog(ERROR, "internal error: demoprot called with a different protocol (%s)",
-						parsed_url->protocol);
+			ereport(ERROR,
```

Comment:
I have a fix on this problem too. https://github.com/cloudberrydb/cloudberrydb/pull/77/files#diff-5fec93245c5c7210063927fcfbce3f57397b7c2b9d1e3a251e5f8513d0011d48

I will cherry-pick this pr first. But how about we use regex to ignore all the case like \(*.c:[0-9]*\) ?